### ISSUE TITLE
persistent_list key will be freed each request (php70)

### ISSUE THREAD

**Issue by guoxiaod** [aerospike/aerospike-client-php#131]
persistent_list is a persistent HashTable, and the key should also be persistent.
in php 7.0, zend_hash_add only assign the key's pointer to HashTable, and addref for it, 
so the key will be freed in RSHUTDOWN, and will be null in NEXT REQUEST.

we should make the key persistent：

```

$ git diff
diff --git a/src/aerospike/aerospike_helper.c b/src/aerospike/aerospike_helper.c
index 5f4d3aa..654ccbf 100644
--- a/src/aerospike/aerospike_helper.c
+++ b/src/aerospike/aerospike_helper.c
@@ -513,7 +513,7 @@ aerospike_helper_object_from_alias_hash(Aerospike_object* as_object_p,
                strlen(alias_to_hash), (void *) &new_le, sizeof(zend_rsrc_list_entry), NULL);
                ((aerospike_ref *) new_le.ptr)->ref_hosts_entry++;
 #else
-               zend_hash_add(persistent_list, zend_string_init(alias_to_hash, strlen(alias_to_hash), 0),
+               zend_hash_add(persistent_list, zend_string_init(alias_to_hash, strlen(alias_to_hash), 1),
                                &new_le);
                ((aerospike_ref *) new_le.value.ptr)->ref_hosts_entry++;
 #endif



```

**Comment by guoxiaod** [comment]
it seems memory leaks in the above code, it should be:

```
zend_string * tmp_key =  zend_string_init(alias_to_hash, strlen(alias_to_hash), 1);
zend_hash_add(persistent_list, tmp_key, &new_le);
zend_string_release(tmp_key);
```

**Comment by jboone100** [comment]
Thanks for tracking this down @guoxiaod. This change will be included in the 3.4.14 release.

**Comment by guoxiaod** [comment]
@jboone100  it seems that all the zend_hash_*  functions should be reviewed for php7.x

**Comment by jboone100** [comment]
Fix rolled into 3.4.14 release and similar code was put in place for the persistent SHM handling.
